depends/CMakeLists.txt

```diff
@@ -28,6 +28,6 @@ include(ExternalProject)
 ExternalProject_Add(
   libsodium
   SOURCE_DIR ${LIBSODIUM_SOURCE_DIR}
-  CONFIGURE_COMMAND cd ${LIBSODIUM_SOURCE_DIR} && ./autogen.sh && ./configure --prefix=<INSTALL_DIR> --enable-minimal --disable-shared --enable-static
+  CONFIGURE_COMMAND cd ${LIBSODIUM_SOURCE_DIR} && DO_NOT_UPDATE_CONFIG_SCRIPTS=1 ./autogen.sh && ./configure --prefix=<INSTALL_DIR> --enable-minimal --disable-shared --enable-static
```

Comment:
Seems like these changes should be contained in #332

Reply:
Ah, I had to rebase onto that branch in order to make the CI work, assuming it would be merged first.  It should disappear if we merge #332.

Reply:
Just rebased on top of develop.